extensions/apicurio-registry-avro/runtime/pom.xml

```diff
@@ -28,7 +28,10 @@
             <groupId>io.apicurio</groupId>
             <artifactId>apicurio-common-rest-client-vertx</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>io.quarkus</groupId>
```

Comment:
That dependency should be optional.  Isn't it @geoand ?

Reply:
Exactly

Reply:
Are you referring to quarkus-kubernetes-service-binding or apicurio-common-rest-client-vertx